test/indexTest.js

```diff
@@ -13,13 +12,20 @@ describe( "index.js", () => {
     beforeEach( () => {
       window.document.body.innerHTML = '<main></main>'
       window.fetch = require( 'node-fetch' );
-      chai.spy.on( window, 'fetch' );
     } );
 
     it( "sends a fetch request to 'https://anapioficeandfire.com/api/books'", async () => {
+      chai.spy.on( window, 'fetch' );
       await fetchBooks()
       expect( window.fetch, "A fetch to the API was not found" )
         .to.have.been.called.with( 'https://anapioficeandfire.com/api/books' );
     } )
+
+    it( "sends renders books onto the DOM'", async () => {
```

Comment:
Maybe change to "...into the DOM by passing a JSON object to `renderBooks`"? Then the failure is more instructive?

Reply:
I have updated the testing language